bedrock/firefox/templates/firefox/android/index.html

```diff
@@ -311,36 +311,28 @@ <h2 class="heading">{{ _('Keep your private info private') }}</h2>
     </p>
 
     <ul>
-      <li id="dnt">
-        <h3>{{ _('Do not track') }}</h3>
-
-        <p>
-        {% trans %}
-          Enable this Firefox innovation to help you stay in control of how your browsing
-          information is collected and used online.
-        {% endtrans %}
-        </p>
-
-        <a href="{{ url('firefox.dnt') }}">{{ _('Learn more about Do Not Track') }}</a>
-      </li>
-      <li id="private-browsing">
-        <h3>{{ _('Private Browsing') }}</h3>
-
-        <p>
-        {% trans %}
-          When activated, Firefox for Android won’t accept cookies, remember passwords or
-          record any sites that you’ve visited.
-        {% endtrans %}
-        </p>
-
-        <a rel="external" href="https://www.youtube.com/watch?v=Ewk0YtFTBec&amp;list=PLA8ABF6B7A66A0C5E&amp;index=15">
-          {% if l10n_has_tag('android_spring_updates') %}
-            {{ _('Learn how to open a private window') }}
-          {% else %}
-            {{ _('See how Private Browsing works') }}
-          {% endif %}
-        </a>
-      </li>
+    {% if l10n_has_tag('tracking_protection') %}
```

Comment:
Again we probably need to use a waffle switch here? Suggest using the same name for the switch as in #3375